packages/core/src/components/service-message/utils/resolveMessageFromServiceMessage.ts

```diff
@@ -0,0 +1,12 @@
+import { ServiceMessage } from "../types";
+
+export const resolveMessageFromServiceMessage = (serviceMessage: ServiceMessage, isError: boolean): { message: string, url?: string | null } => {
+    if (isError) return {
+        message: "Could not retrieve service message.",
```

Comment:
Do we want a service message informing the user that there was an error fetching the service message? Should this not be added as a toast in the fetcher instead?

Reply:
I agree, however we do not have a toast system yet. I think for now this solution is acceptable, but we can create an issue on displaying a toast in the future?

Reply:
Sure thing 🆗

Reply:
Issue created https://github.com/equinor/mad/issues/320